README.md

```diff
@@ -72,19 +90,101 @@ Components in beta stage may include breaking changes, new features, and bug fix
 | Internet Explorer           | 10 & 11 |
 | Safari & Mobile Safari      | Current |
 
+<hr>
+<!--  -->
+
+## Getting Started
+
+### Dependencies
+-   D3
+-   Polyfills:
+    -   Object.assign
+    -   Object.values
+
+### Install
+
+You can install Carbon via `npm` or download the [latest release](https://github.com/cerner/terra-graphs/releases/latest) from GitHub.
+
+```sh
+npm i @cerner/carbon-graphs --save-dev
+```
+
+<!-- ```sh
+npm i @cerner/Terra Graphs-graphs --save-dev
+``` -->
+
+### Getting Started
+
+-   [Getting Started](./packages/terra-graphs/docs/getting-started/GettingStarted.md)
+-   [API Reference](docs/README.md)
```

Comment:
The link is a relative local link so it will work anywhere on any branch.. I tested it in rich text view and it works. The link you shared points to the migration branch, so it won't work on main.